src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  --head-marg: 50px;
+  --max-header-wight: 1200px;
+  --min-header-wight: 1024px;
```

Comment:
The CSS variable `--max-header-wight` seems to be a typo; it should be `--max-header-width`. The same typo is present in `--min-header-wight`. Remember that correct naming is crucial for code readability and maintainability.